test/hotspot/jtreg/runtime/cds/appcds/BadBSM.java

```diff
@@ -42,7 +42,8 @@ public static void main(String[] args) throws Exception {
 
     OutputAnalyzer out = TestCommon.dump(appJar,
         TestCommon.list("WrongBSM",
-                        "@lambda-proxy WrongBSM 7"));
+                        "@lambda-proxy WrongBSM 7"),
+        "-Xlog:cds+lambda=debug");
```

Comment:
Please align line 46 with 45.

Reply:
The `"-Xlog:cds+lambda=debug"` parameter is actually at the same level as the `TestCommon.list("WrongBSM",...)` parameter. Aligning it with line 45 would make it look like an argument for the `TestCommon.list()` call.